models/velodrome/optimism/velodrome_optimism_trades.sql

```diff
@@ -52,14 +81,14 @@ SELECT
         when lower(erc20a.symbol) > lower(erc20b.symbol) then concat(erc20b.symbol, '-', erc20a.symbol)
         else concat(erc20a.symbol, '-', erc20b.symbol)
     end as token_pair
-    ,dexs.token_bought_amount_raw / power(10, erc20a.decimals) AS token_bought_amount
-    ,dexs.token_sold_amount_raw / power(10, erc20b.decimals) AS token_sold_amount
-    ,dexs.token_bought_amount_raw  AS token_bought_amount_raw
-    ,dexs.token_sold_amount_raw  AS token_sold_amount_raw
+    ,cast(dexs.token_bought_amount_raw as double) / cast( power(10, erc20a.decimals) as double) AS token_bought_amount
+    ,cast(dexs.token_sold_amount_raw as double) / cast( power(10, erc20b.decimals) as double) AS token_sold_amount
+    ,CAST(dexs.token_bought_amount_raw AS double) AS token_bought_amount_raw
+    ,CAST(dexs.token_sold_amount_raw AS double) AS token_sold_amount_raw
```

Comment:
we shouldn't cast these, unless necessary to get source types to `uint256`, which is what we use for raw amounts in `dex.trades`